test/flint/fmpz_mod-test.jl

```diff
@@ -128,6 +131,21 @@ end
   @test_throws Exception R6(R22(1))
   @test_throws Exception R2(R3(1))
   @test_throws Exception R3(R2(1))
+
+  ZZmod720,_ = residue_ring(ZZ,720)
+  ZZmodZZ720,_ = residue_ring(ZZ,ZZ(720))
+  @test is_nilpotent(ZZmod720(30))
+  @test is_nilpotent(ZZmodZZ720(30))
+  nilp720 = filter((r -> is_nilpotent(ZZmod720(r))), 0:719);
+  @test length(nilp720) == 24  # 720/(2*3*5)
```

Comment:
Better expressed like so:
```suggestion
  @test count(is_nilpotent, ZZmod720) == 24  # 720/(2*3*5)
```

Reply:
Your suggestion to use `count` works in an `Oscar` context but not in an `AbstractAlgebra` context because the following error arises:
```
julia> count(is_nilpotent, ZZmod720)
ERROR: MethodError: no method matching iterate(::EuclideanRingResidueRing{BigInt})
```
We could leave the test as is (_i.e._ using `filter`), or we could implement `iterate` for `EuclideanRingResidueRing{BigInt}`.  Which is the preferred/better option?

Reply:
It doesn't work in AA, true, but it works fine in Nemo for me, and this PR is targetting Nemo, isn't it?